src/display/editor/ink.js

```diff
@@ -654,6 +658,55 @@ class InkEditor extends AnnotationEditor {
     this.enableEditMode();
   }
 
+  /**
+   * Track a new pointer
```

Comment:
Here, and everywhere else, please remember that sentences should always end with a period.